pkg/sql/schema_changer_test.go

```diff
@@ -7006,6 +7006,7 @@ func TestCheckConstraintDropAndColumn(t *testing.T) {
 				jobControlMu.Lock()
 				scJob, err := s.JobRegistry().(*jobs.Registry).LoadJob(ctx, jobID)
 				if err != nil {
+					jobControlMu.Unlock()
```

Comment:
nit: consider adding a quick note about the usage of the mutex below? At first glance, I was confused about why this wasn't a `defer jobControlMu.Unlock()`.

Reply:
Done.